src/classes/project_data.py

```diff
@@ -906,42 +906,64 @@ def check_if_paths_are_valid(self):
 
         from PyQt5.QtWidgets import QFileDialog, QMessageBox
 
+        log.info("checking project files...")
+
         # Loop through each files (in reverse order)
         for file in reversed(self._data["files"]):
             path = file["path"]
             parent_path, file_name_with_ext = os.path.split(path)
+
+            log.info("checking file %s" % path)
             while not os.path.exists(path) and "%" not in path:
                 # File already exists! Prompt user to find missing file
-                QMessageBox.warning(None, _("Missing File (%s)") % file["id"], _("%s cannot be found.") % file_name_with_ext)
-                starting_folder = QFileDialog.getExistingDirectory(None, _("Find directory that contains: %s" % file_name_with_ext), starting_folder)
-                log.info("Missing folder chosen by user: %s" % starting_folder)
-                if starting_folder:
-                    # Update file path and import_path
-                    path = os.path.join(starting_folder, file_name_with_ext)
+                # try to find clip with previous starting folder:
+                if starting_folder and os.path.exists(os.path.join(starting_folder, file_name_with_ext)):
+                    # Update clip path
                     file["path"] = path
                     get_app().updates.update(["import_path"], os.path.dirname(path))
-                else:
-                    log.info('Removed missing file: %s' % file_name_with_ext)
-                    self._data["files"].remove(file)
+                    log.info("Auto-updated missing file: %s" % file["path"])
```

Comment:
It actually looks like this line might be the _only_ conflict here (it's certainly one of the only) — and it only conflicts because my commit changed it to
```python3
log.info("Auto-updated missing file: %s" % path)
```
whereas @fschill changed it to
```python3
log.info("Auto-updated missing file: %s" % file["path"])
```
...however, those lines are _literally_ equivalent, just two lines earlier we set `file["path"] = path`!

So, it does not matter the _tiniest_ but which version you go with. If that line is the only conflict, then it can be resolved and merged. (If not, I guess let me know what the other conflicts are? I don't appear to have visibility into that, as a non-collaborator non-submitter. ...Or if I do, and I'm just not seeing how, let me know how I can look over the other conflicts.)